### PR TITLE
Added Utils file + Updated Parser

### PR DESCRIPTION
**Pull request by OfficialNoob**
This is my version of the Utils
There is also https://github.com/ajayyy/SponsorBlock/pull/85

### CODE REVIEW

**Comment by ajayyy** [comment]
Perfect, exactly what I was looking for

**Comment by OfficialNoob** [comment]
Current: https://github.com/ajayyy/SponsorBlock/blob/b964d93ea948b94515a7a93dc573c5ab5a52be3c/utils.js

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function getYouTubeVideoID(url) {
+
+    try { // Attempt to parse url
+        var obj = new URL(url);
+    } catch (e) {      
+        console.error("[SB] Unable to parser URL");
```
Comment (ajayyy) [inline utils.js]: Fix typo here

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return false
```
Comment (ajayyy) [inline utils.js]: Add semicolon please

**Pull request merged**